content/community/organization/policies.md

```diff
@@ -0,0 +1,110 @@
+
+## Trademarks and Copyrights
+
+### Trademark Policy
+
+[Haiku, Inc](https://haiku-inc.org). owns the 'Haiku'® name, HAIKU logo®, HAIKU Background Leaf™, and HAIKU Leaf™ (registered) trademarks. As an open source project, the name and brand that is associated with the Project is vital to the Project's reputation and the sense of familiarity that end-users expect from the Software. Anyone is able to freely use the code that comprises Haiku, however the trademarks of Haiku cannot freely be used in the same liberal manner.
+
+The [Trademark Policy](/about/trademark_policy) page on the "about" section of this website contains the latest policy revision.
+
+### Haiku Distribution Guidelines
+
+The Haiku project believes that having one distribution (the one officially released by the project) is the best long term strategy to ensure success of the operating system.  For this reason anyone who is considering creating a new distribution should think long and hard before doing so.
+
+The [Haiku Distro Guidelines](/community/guidelines_creating_haiku_distribution) page on the "about" section of this website contains the latest policy revision regarding the creation of a Haiku distribution.
+
+## Software
+
+### General Strategy for Releases
+
+As the Haiku project is dominated by volunteer effort, the project is not able to commit to fixed deadlines because it is not possible to ascertain the level of contributions available over a period of time and it is also not possible to direct any of the development team to any specific task.  The team is self-organizing and each individual applies themself to the project as they see fit or as they wish.  For this reason each build -- be that ad-hoc, nightly or a formal release -- contains the appropriate contributions from the contributors to that point in time.
+
+### Nightly Images
+
+Haiku's nightly images are a build of the entire operating system that contains the latest enhancements, bug fixes, and hardware support. Please keep in mind that there may also be regressions and broken features in these builds which have not yet been identified or resolved. Users should only use a nightly build at their own risk and be aware of the potential for experiencing issues.
+
+### Formal Releases
+
+The Haiku project aims to make an annual release of the operating system.  In general a formal release will tend to be made in the northern hemisphere spring / southern hemisphere autumn.  The rationale for this timing is because it is the time of year at which the development team are most likely to have capacity to deal with a release.  At the release time, the development team will attempt to stabilize the state of the project and to create a reference point for users of the operating system.  A release may be deferred or may be cancelled entirely if there are difficulties making the release for any reason.
```

Comment:
I don't think this should go in the policies either. As I mentionned on the mailing list, some things are unwritten because they allow us to change our mind on how we do things. I think we should move this to a separate page and probably put it in the Trac Wiki to emphasize that it is editable by any contributor.